support/crown/src/main.rs

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#![feature(rustc_private)]
+#![warn(rustc::internal)]
+#![allow(rustc::potential_query_instability)]
```

Comment:
Will we be able to compile crown with a nightly and then be able to compile Servo itself with a stable version of rust?

Reply:
I think crown need to be compiled with (roughly) same version of rust, so we would likely need to use nightly features on stable trick. I need to investigate how clippy, klint or any other drivers handle this.

Reply:
dylint runs with same rust version that lint was compiled with so this a no go for us

source: https://github.com/trailofbits/dylint/blob/master/docs/how_dylint_works.md#limitations

Reply:
I think a reasonable plan here might be to compile crown with RUSTC_BOOTSTRAP turned on.